.circleci/config.yml

```diff
@@ -177,15 +177,15 @@ commands:
       - run:
           name: Run cucumber tests
           command: |
-            TESFILES=$(circleci tests glob "features/**/*.feature" | circleci tests split --split-by=timings --timings-type=filename)
+            TESTFILES=$(circleci tests glob "features/**/*.feature" | circleci tests split --split-by=timings --timings-type=filename)
 
             RUBYOPT='-W0' bundle exec cucumber \
             --format pretty \
             --format json \
             --out /tmp/test-results/cucumber/tests.cucumber \
-            -- ${TESFILES}
+            -- features/provider_management.feature ${TESTFILES}
```

Comment:
Was this change meant to stay in or just as a test?

Reply:
just testing on circle